test/README.md

```diff
@@ -85,7 +102,111 @@ mosquitto -d -p 1883
 nohup ./edge_core > edge_core.log 2>&1 &
 ```
 
+### Verify Device test
```

Comment:
Can we have a structure like this : 

### Verify

#### Add Device

#### Remove Device

#### Add Pod


#### Remove Pod